docs/howtos/audit-scanner.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: "Audit Scanner Installation"
+title: "Audit Scanner Installation"
+---
+
+# Audit Scanner installation
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+## Installation
+
+1. Install the `kubewarden-crds` Helm chart, ensuring you have app version
+   `v1.7.0` or higher. Remember to enable the audit scanner component.
+
+   ```console
+   helm install kubewarden-crds kubewarden/kubewarden-crds
+   ```
+   :::caution
+      The PolicyReport Custom Resource Definitions must be available to store the
```

Comment:
This reads like an overexplanation: "if you don't install them then they don't get installed".
Maybe we can change it to provide reasons. e.g:

> Caution: in case that the needed PolicyReport CRDs are already present in the cluster, one will not be able to install them via the `kubewarden-crds` chart. One can then set `installPolicyReportCRDs` to `false` for the chart, and then the Kubewarden stack will not own those CRDs, but the admin.